stable/kured/templates/unlock-cronjob.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if .Values.autolock.enabled }}
```

Comment:
Could make the two cronjobs use the same template (there's quite a bit of repetition between them, and they are "twins", so their sameness is inherent) which might be tidier, but might be harder to understand.

Reply:
Indeed but I tried but I don't know how to do that, using functions in .tpl  file storing the whole template ?

Reply:
You can do this with something like:
```yaml
{{ $root := . }}
{{- range $direction := list "lock" "unlock" }}
apiVersion: batch/v1beta1
kind: CronJob
metadata:
  name: {{ template "kured.fullname" $root }}-{{ $direction }}
...
{{- if eq $direction "lock" }}
  schedule: {{ $root.Values.autolock.scheduleLock | quote }}
{{- else }}
  schedule: {{ $root.Values.autolock.scheduleUnlock | quote }}
{{- end }}
...
{{- end }}
```
But I'm inclined to say get it merged as it is, then we can worry about dry-ing up the templates later.